examples/aishell/s0/run.sh

```diff
@@ -27,11 +27,10 @@ train_config=conf/train_conformer.yaml
 checkpoint=
 cmvn=true
 dir=exp/sp_spec_aug
-
 # use average_checkpoint will get better result
 average_checkpoint=true
 decode_checkpoint=$dir/final.pt
-average_num=5
+average_num=10
```

Comment:
why 10? the larger the better? what about 15?

Reply:
10 can help get a more stable results